packages/package_info_plus/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 1.0.4
 
-* Update minimum Flutter and Dart version to 3.13 and 3.1.
```

Comment:
This log should be kept because it occurred between 1.0.3 and 1.0.4.
refer to https://github.com/flutter-tizen/plugins/blob/master/packages/video_player_avplay/CHANGELOG.md#041

Reply:
Sorry for the misunderstanding - thank you for the adaptations